docs-kits/kits/behaviour-twin-kit/adoption-view/terminology.md

```diff
@@ -0,0 +1,78 @@
+﻿---
+id: terminology
```

Comment:
@jSchuetz88 : Input for the overarching glossary! 
https://catenax-ev.github.io/docs/next/working-model/glossary